lib/MySQL_Session.cpp

```diff
@@ -2433,22 +2428,21 @@ bool MySQL_Session::handler_again___status_SETTING_MULTI_STMT(int *_rc) {
 		if (rc==-1) {
 			// the command failed
 			int myerr=mysql_errno(myconn->mysql);
-			// if rc was '-1' but 'mysql_errno' is 0, it means that the connection
```

Comment:
Instead of replacing this with 3 calls, we can use this:
```
MyHGM->p_update_mysql_error_counter(p_mysql_error_type::mysql, myconn->parent->myhgc->hid, myconn->parent->address, myconn->parent->port, ( mysql_errno(myconn->mysql) ? mysql_errno(myconn->mysql) : ER_PROXYSQL_OFFLINE_SRV ) );
```